src/subprocess-win32.cc

```diff
@@ -81,13 +81,13 @@ bool Subprocess::Start(SubprocessSet* set, const string& command) {
   security_attributes.bInheritHandle = TRUE;
   // Must be inheritable so subprocesses can dup to children.
   HANDLE nul =
-      CreateFileA("NUL", GENERIC_READ,
+      CreateFile(L"NUL", GENERIC_READ,
```

Comment:
I recommend creating a new variable

static const wchar_t nulstr[] = L"NUL";

to make it very clear that this intended to be a wchar_t. Casual readers may miss the significance of the L.

Reply:
shouldn't there be an error if someone passes a char* as wchar_t*?

Reply:
Apparently not, if the -municode flag is dropped.

My suggestion here is for the sake of human readers, not code safety.

Reply:
`-municode` flag won't be dropped.

Reply:
You say that now...

If someone *does* compile ninja without that flag, the wrong function would be called.

And in the future things might change in other ways.

Regardless, I made my suggestion, The "L" symbol is easy to overlook.

I would also be satisfied by the call being to `CreateFileW` instead.

Reply:
> I would also be satisfied by the call being to `CreateFileW` instead.

+1, and then no `nulstr` variable please.